stories/link-bars/ScrollArrowsGradientBackgroundStory.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { storiesOf, action, linkTo } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies,no-unused-vars
+import { DemoContainer } from '../storybook-components';
+
+import { Heading, HugeHeading } from '../../src/atoms/Heading';
+import { Code } from '../../src/atoms/Code';
+
+// HorizontalLinkBar
+import {
+	ScrollArrowsLinkBar,
+	LinkBarLink,
+} from '../../src';
+
+import theme from '../../src/themes/default-theme';
+
+const { colors } = theme;
+
+const ScrollArrowsGradientBackground = () => (
+	<section>
+		<HugeHeading>ScrollArrowsLinkBar</HugeHeading>
+		<p>... which have gradient background</p>
+		<p>A ScrollArrowsLinkBar is a HorizontalLinkBar with clickable arrows for scrolling.</p>
+
+		<Heading>Demo</Heading>
+		<DemoContainer>
+			<ScrollArrowsLinkBar
+				background={`linear-gradient(0deg, ${colors.grayTint}, ${colors.white} .1rem)`}
+				shouldAdjustForNestedPadding
+				drawRightArrowInitially
+				arrowGradientBackground={`${colors.white}`}
+				arrowGradientHeight="97%"
```

Comment:
I am not happy with giving a value of 97% here, but it works.  It should have been .1rem, but then I also would have to change the css from top:0 to bottom:0.  We should improve on this later.